build.gradle

```diff
@@ -29,6 +29,7 @@ repositories {
 		url = "https://maven.meteordev.org/releases"
 	}
 	maven {url "https://jitpack.io"}
+	maven { url "https://repo.codemc.io/repository/maven-public/" } // For Occlusion Culling library
```

Comment:
Please place this above jitpack. Jitpack should always be placed last. Mentioned in https://jitpack.io/docs/:

> Note: when using multiple repositories in build.gradle it is recommended to add JitPack at the end. Gradle will go through all repositories in order until it finds a dependency.